CHANGELOG.md

```diff
@@ -0,0 +1,1104 @@
+  * Fully adopts upstream commaai’s openpilot, opendbc (car interface and safety), and panda test suites to ensure consistent safety compliance and reliability across all systems
+  * Added regression testing to verify expected behavior and maintain stability across core modules
+  * Aligns with comma.ai’s safety policy: preserving driver monitoring, actuation checks, and safety test suite coverage
+  * Some features have not yet been reimplemented in this rewrite and are temporarily disabled in this release. They will return in a future version once fully ported and validated:
+    * Visuals: Rocket Fuel
+    * Visuals: Displaying Braking Status
+    * Vehicle: Toyota - Enforce Stock Longitudinal Control
+    * Subaru: Increase Steering Torque
+    * Longitudinal: Acceleration Personality
+    * UI: Display CPU Temperature on Sidebar
+    * Lateral: Block Lane Change with Road Edge Detection
+    * UI: Display DM Camera in Reverse Gear
+    * UI: Auto-hide Selected UI Elements
+    * Visuals: Display End-to-End Longitudinal Status
+    * Toyota: Stop and Go Hack (alpha)
+    * Visuals: Onroad Settings
+    * Honda: Serial Steering Support
+    * Volkswagen: Non-ACC Platforms Support
+    * Longitudinal: Dynamic Personality
+    * Honda Nidec: Allow Stock Longitudinal Control
+    * Lateral Planner: Dynamic Lane Profile
+    * Lateral: Custom Camera and Path Offsets
+    * Toyota: Door Controls
+* Major Features & Systems
+  * Modular Assistive Driving System (MADS)
+    * Complete driving assistance framework
+  * Driving Model Manager
+    * Custom driving model selection with support for about 86 models (as of writing), from Optimus Prime (September 2023) up to The Cool People’s Models (October 2025)
+  * Neural Network Lateral Control (NNLC)
+    * Advanced torque-based lateral control
+  * Dynamic Experimental Control (DEC)
+    * Intelligent longitudinal control adaptation
+  * Speed Limit Assist (SLA)
+    * Comprehensive speed limit integration featuring `mapd` for offline map limits downloads, a Speed Limit Resolver for sourcing data (from car, map, combined, etc), on-screen UI for Speed Limit Information/Warning, and Speed Limit Assist (SLA) to adjust cruise speed automatically.
+  * Intelligent Cruise Button Management (ICBM)
+    * System designed to manage the vehicle’s speed by sending cruise control button commands to the car’s ECU.
+  * Smart Cruise Control Map & Vision (SCC-M / SCC-V)
+    * When using any form of long control (openpilot longitudinal or **ICBM**) it will control the speed at which you enter and perform a turn by leveraging map data (**SCC-M**) and/or by leveraging what the model sees about the curve ahead (**SCC-V**)
+  * Vehicle Selector
+    * If your vehicle isn’t fingerprinted automatically, you can still use the vehicle selector to get it working
+  * sunnylink Integration
+    * Cloud connectivity and settings backup/restore
```

Comment:
I'd also maybe mention that it's ready for remote setting management and remote model switching (pending to release on dashboard)